### PR TITLE
Changed hyperparam configuration in the DQN tutorial

### LINKED ISSUE #3080: [Improve] - Pytorch Reinforcement DQN Tutorial
### 🚀 Describe the improvement or the new tutorial

In current tutorial DQN is not performing well even after 600 episodes. I would like to update hyper parameters, so it reaches optimal policy by the end of training.

Following parameters needs to change and need to replace the Result image.

EPS_START = 1.0
EPS_END = 0
EPS_DECAY = 1500
LR = 5e-4

### Existing tutorials on this topic

_No response_

### Additional context

image

cc @vmoens @nairbv

### PR DESCRIPTION
**Pull request by splion-360**
Fixes #3080 

## Description
- This PR updates the hyper parameters for the `CartPole-v1` environment in the DQN tutorial to better match the results shown in the reference image.
- A fixed `seed` has been added to ensure reproducibility of training behavior and evaluation outcomes.  

## Checklist

- [x] The issue that is being fixed is referred in the description (see above "Fixes #3080 ")
- [x] Only one issue is addressed in this pull request
- [x] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Comment by svekars** [comment]
I believe the only changes should be in the actual DQN tutorial and not in all the other files.

**Comment by splion-360** [comment]
Thank you @svekars for the feedback. I will work on it

**Comment by AlannaBurke** [comment]
Any updates to this?
